src/main/kotlin/lotto/domain/WinningLotto.kt

```diff
@@ -0,0 +1,12 @@
+package lotto.domain
+
+data class WinningLotto(val lotto: Lotto, val bonusNumber: LottoNumber) {
+
+    init {
+        require(
+            (bonusNumber in lotto).not()
```

Comment:
`!in`과 같이 코드를 작성할 수도 있습니다.
```suggestion
            bonusNumber !in lotto
```